_pages/en_US/installing-boot9strap-(homebrew-launcher).txt

```diff
@@ -15,13 +15,17 @@ To use the [magnet](https://en.wikipedia.org/wiki/Magnet_URI_scheme) links on th
 * The latest release of [safehax](https://github.com/TiniVi/safehax/releases/latest)
 * The latest release of [udsploit](https://github.com/smealum/udsploit/releases/latest)
 * The latest release of [Luma3DS](https://github.com/AuroraWright/Luma3DS/releases/latest) *(the `.7z` file)*
+* The latest release of [GodMode9](https://github.com/d0k3/GodMode9/releases/latest)
 
 #### Instructions
 
 ##### Section I - Prep Work
 
 1. Remove your SD card from your device, leaving it on the Homebrew Launcher, then insert your SD card into your computer
 1. Copy `boot.firm` from the Luma3DS `.7z` to the root of your SD card
+1. Create a folder named `luma` on the root of your SD card
+1. Create a folder named `payloads` in the `luma` folder on your SD card
+1. Copy `GodMode9.firm` from the GodMode9 `.zip` to the `/luma/payloads/` folder on your SD card
```

Comment:
There's no need to do this now. If you do this now, it'd have to be included in all of the other Installing Boot9Strap pages...